### ISSUE TITLE
No Start Menu entry or Desktop Shortcut created on Windows installer.

### ISSUE THREAD

**Issue by kruug** [LibreCAD/LibreCAD#966]
### Expected behavior
Entry created in Start Menu and/or Desktop.

### Observed behavior
No entry created.

### Steps to reproduce or sample file
Install on Windows.

### Operating System and LibreCAD version info
Windows 10 Enterprise x64
LibreCAD 2.1.3 and 2.2.0-rc1

**Comment by doudz** [comment]
Same with 2.2.0-rc2

It's a little thing important to add.
Company users just suppose the software is missing because they can't find it in Windows Start menu

**Comment by InnerBushman** [comment]
I agree. This is very important to me too. When installing it on our company machines I have to explain my colleagues that they have to manually create the shortcut on their desktop themselves. It's not only very annoying but also makes me look unprofessional. 

What would it take to add it to the installer? I never created one before.
